test/test_ops.py

```diff
@@ -1073,7 +1072,6 @@ def test_bbox_xywh_cxcywh(self):
             [[0, 0, 100, 100], [0, 0, 0, 0], [10, 15, 20, 20], [23, 35, 70, 60]], dtype=torch.float
         )
 
-        # This is wrong
```

Comment:
Really? It isn't. I verified it. I don't know how this comment is here. (Or is it probably me who left it 2 years ago lol)

Reply:
Yup, see #2710 :sweat_smile: If you verified that this ok, we can remove.